build/files.js

```diff
@@ -180,11 +180,12 @@ const headRegexp = /(^module.exports = \w+;?)/m
       /(?:var|const) Buffer = require\('buffer'\)\.Buffer;/
     , `/*<replacement>*/
   const Buffer = require('safe-buffer').Buffer
+  const OurUint8Array = global.Uint8Array || function () {}
```

Comment:
Another thing: Use `var` instead of `const` here (and in the other files below), to ensure that the `instanceof` optimization definitely triggers.